src/mainframe/endpoints/report.py

```diff
@@ -188,31 +189,51 @@ def report_package(
                     f"`{name}@{version}` has no matched rules in the database"
                 ),
             )
-            log.debug("Missing additional_information field", tag="missing_additional_information")
+            log.error(
+                "Missing additional_information field", error_message=error.detail, tag="missing_additional_information"
+            )
             raise error
 
     rules_matched.extend(rule.name for rule in row.rules)
 
     additional_information = body.additional_information
 
-    send_report_email(
-        graph_client,  # type: ignore
-        recipient=body.recipient or mainframe_settings.email_recipient,
-        package_name=name,
-        package_version=version,
-        inspector_url=inspector_url,
-        additional_information=additional_information,
-        rules_matched=rules_matched,
-    )
+    if body.use_email is True:
+        send_report_email(
+            graph_client,  # type: ignore
+            recipient=body.recipient or mainframe_settings.email_recipient,
+            package_name=name,
+            package_version=version,
+            inspector_url=inspector_url,
+            additional_information=additional_information,
+            rules_matched=rules_matched,
+        )
+    else:
+        if additional_information is None:
+            # summary field is required
+            error = HTTPException(
+                400,
+                detail="additional_information is required when using Observation API",
+            )
+            log.error(
+                "Missing additional_information field", error_message=error.detail, tag="missing_additional_information"
+            )
+            raise error
+
+        to_send = dict(
+            kind="is_malware", summary=additional_information, inspector_url=inspector_url, yara_rules=rules_matched
+        )
```

Comment:
`yara_rules` must go in the `extras` dict according to the model:
```suggestion
        to_send = dict(
            kind="is_malware", summary=additional_information, inspector_url=inspector_url, extras=dict(yara_rules=rules_matched)
        )
```